src/components/Clicker.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
You can import it like `import React, { Component } from 'react'`
then, you can just write 

`class Clicker extends Component`